### PR TITLE
fix: success dialog popping wrong

### PR DESCRIPTION
**Pull request by Cristhianzl**
🐛 (sideBarFolderButtons/index.tsx): fix success message not being set when files array is empty

### CODE REVIEW

**Review by lucaseduoli — CHANGES_REQUESTED** [review]
Check if the files are empty before trying to call uploadFlow

File: src/frontend/src/components/folderSidebarComponent/components/sideBarFolderButtons/index.tsx [hunk 1]
```diff
@@ -67,6 +67,10 @@ const SideBarFoldersButtonsComponent = ({
 
   const handleUploadFlowsToFolder = () => {
     createFileUpload().then((files: File[]) => {
+      if (files?.length === 0) {
+        return;
```
Comment (ogabrielluiz) [inline src/frontend/src/components/folderSidebarComponent/components/sideBarFolderButtons/index.tsx:71]: Maybe set notice

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]

File: src/frontend/src/components/folderSidebarComponent/components/sideBarFolderButtons/index.tsx (hunk 1 above)
Comment (Cristhianzl) [reply src/frontend/src/components/folderSidebarComponent/components/sideBarFolderButtons/index.tsx:71]: I think this is not the case.. the user just pressed "Cancel" and the popup was been activated

**Review by lucaseduoli — APPROVED** [review]
LGTM

File: src/frontend/src/components/folderSidebarComponent/components/sideBarFolderButtons/index.tsx (hunk 1 above)
Comment (ogabrielluiz) [reply src/frontend/src/components/folderSidebarComponent/components/sideBarFolderButtons/index.tsx:71]: I see

**Review by ogabrielluiz — APPROVED** [review]

**Pull request merged**